File: Laboratorio02/CodigoC++/Persona.cpp
#include "Persona.h"

#include <string>
using namespace std;

Persona::Persona(string nombre, int edad) : nombre(nombre), edad(edad) {}

string Persona::getNombre() const {
    return nombre;
}

void Persona::setNombre(string nombre) {
    this->nombre = nombre;
}

int Persona::getEdad() const {
    return edad;
}

void Persona::setEdad(int edad) {
    this->edad = edad;
}

string Persona::toString() const {
    return "Persona: " + nombre + " ; edad: " + to_string(edad);
}

File: Laboratorio02/CodigoPython/persona.py
class Persona:
    def __init__(self, nombre: str, edad: int):
        self.nombre = nombre
        self.edad = edad

    @property
    def nombre(self):
        return self._nombre

    @nombre.setter
    def nombre(self, valor):
        self._nombre = valor

    @property
    def edad(self):
        return self._edad

    @edad.setter
    def edad(self, valor):
        self._edad = valor

    def __str__(self):
        return f"Persona: {self.nombre} ; edad: {self.edad}"
